mtp_cashbook/templates/cashbook/transaction_batch_list.html

```diff
@@ -1,6 +1,8 @@
 {% extends "base.html" %}
 {% load currency core i18n %}
 
+{% block template_type %}v-batch{% endblock %}
```

Comment:
Yes, I've used it previously and it's similar to conventions like using `u-` for a [utility](https://github.com/ministryofjustice/govuk-template/blob/master/assets/scss/govuk/_utilities.scss).